cmd/video.go

```diff
@@ -15,6 +16,23 @@ var videoCmd = &cobra.Command{
 	Short: "Import a video by ID",
 	Long:  `Import a YouTube video by ID and assign it to a creator.`,
 	Run: func(cmd *cobra.Command, args []string) {
+		if id == "" && url == "" {
+			log.Fatal("command must include either video ID or URL")
+		}
+		if id != "" && url != "" {
+			log.Fatal("both video ID and URL provided, expected only one")
+		}
+
+		if url != "" {
+			re := regexp.MustCompile(`(?:v|embed|watch\?v)(?:=|/)([^"&?/=%]{11})`)
```

Comment:
Another valid format is `https://youtu.be/rURhJr0cDnk` and it's been used in a few submissions so far.

I'd be happy for us to have limitations on acceptable formats for simplicity sake, eg:

- https://www.youtu.be/rURhJr0cDnk
- https://www.youtube.com/embed/rURhJr0cDnk
- https://www.youtube.com/watch?v=xspEtjnSfQA

Perhaps the regex is something along the lines of v=XXX or final slash and everything after.

Say I tried https://youtu.be/rURhJr0cDnk?rofl=lmao I would be happy for the parser to fail.